x-pack/plugins/ml/public/application/components/ml_inference/components/additional_advanced_settings.tsx

```diff
@@ -0,0 +1,188 @@
+                  values={{
+                    painlessDocs: (
+                      <EuiLink
+                        external
+                        target="_blank"
+                        href={links.scriptedFields.painlessWalkthrough}
+                      >
+                        Painless
+                      </EuiLink>
+                    ),
+                  }}
+                />
+              }
+            >
+              <EuiTextArea
+                aria-label={i18n.translate(
+                  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.conditionAriaLabel',
+                  { defaultMessage: 'Optional condition for running the processor' }
+                )}
+                value={additionalSettings?.condition ?? ''}
+                onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) =>
+                  handleAdditionalSettingsChange({ condition: e.target.value })
+                }
+              />
+            </EuiFormRow>
+          </EuiFlexItem>
+          {/* TAG AND IGNORE FAILURE */}
+          <EuiFlexItem>
+            <EuiFlexGroup direction="column">
+              <EuiFlexItem>
+                <EuiFormRow
+                  fullWidth
+                  label={
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.TagLabel"
+                      defaultMessage="Tag (optional)"
+                    />
+                  }
+                  helpText={
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.tagHelpText"
+                      defaultMessage="Identifier for the processor. Useful for debugging and metrics."
+                    />
+                  }
+                >
+                  <EuiFieldText
+                    value={additionalSettings?.tag ?? ''}
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                      handleAdditionalSettingsChange({ tag: e.target.value })
+                    }
+                    aria-label={i18n.translate(
+                      'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.tagAriaLabel',
+                      { defaultMessage: 'Optional tag identifier for the processor' }
+                    )}
+                  />
+                </EuiFormRow>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiFormRow
+                  fullWidth
+                  helpText={
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.ignoreFailureHelpText"
+                      defaultMessage="Ignore failures for this processor."
```

Comment:
I would place it more prominently in the configuration. Quite often, the input data fields change, which results in failure. Where do those failed docs go? It would be good to educate the user about their options.

Reply:
Updated in 4b1791c5523f98483b24f0ca0dd3f1e88b855a50 - the change is reflected in the new screenshots.
This doesn't feel done yet though - could you share more of what info we should be showing the user?
For example, should we also be showing the on_failure option here? Or mentioning/linking to it?
I'd be grateful for some feedback on this bit 😄 Thanks!

Reply:
Since we want to create a curated experience for deploying a trained model, I would call this option: "Ingest problematic documents" with explanation text: "If the model fails to produce a prediction, e.g., due to the data schema change, the document will be ingested without the prediction." This option can be activated by default and hidden in "Advanced settings". IMO, "ignore failures" is a double negative, which can confuse users.

 Now, `on_failure` option is actually more important an should be visible in the main configuration. IMO, by default, on failure trigger should be active, as this will tell the users that something is wrong with their data (or their model deployment). I would suggest a default configurations along the following lines:
 ```json
"on_failure": [
    {
      "set": {
        "description": "Index document to 'failed-<index>'",
        "field": "_index",
        "value": "failed-{{{ _index }}}"
      }
    },
   {
      "set": {
        "field": "event.created",
        "value": "{{{ _ingest.timestamp }}}"
      }
    },
    {
      "set": {
        "field": "event.failure.message",
        "value": "{{{ _ingest.on_failure_message }}}"
      }
    },
    {
      "set": {
        "field": "event.failure.processor_type",
        "value": "{{{ _ingest.on_failure_processor_type }}}"
      }
    },
    {
      "set": {
        "field": "event.failure.processor_tag",
        "value": "{{{ _ingest.on_failure_processor_tag }}}"
      }
    },
   {
     "set": {
        "value": "event.failure.pipeline",
        "value": {{ _ingest.on_failure_pipeline }}
     }
   }
  ]
 ```

Reply:
Thanks for the context, @valeriy42! 🙏 
After chatting with @peteharverson, we decided a separate step for failure handling was warranted. I added that step along with the default settings and text you suggested in 64ba3ac4ec279eb78f5f8a97d01005991ae76219

I've updated the step pics in the description and would love your feedback on the new copy and flow when you get a chance.